src/modules/70-pipeline/components/PipelineSteps/Steps/ServiceNowUpdate/ServiceNowUpdateStepMode.tsx

```diff
@@ -437,33 +465,106 @@ function FormContent({
             }}
           />
         </div>
+        {formik.values.spec.ticketType === 'change_task' && CDS_NG_UPDATE_MULTIPLE_SNOW_CHANGE_REQUEST ? (
+          <>
+            <div className={cx(stepCss.formGroup, stepCss.sm)}>
+              <FormInput.CheckBox
+                name="spec.updateMultipleFlag"
+                label={getString('pipeline.serviceNowApprovalStep.updateMultiple')}
+              />
+            </div>
+            <div className={cx(stepCss.formGroup, stepCss.lg)}>
+              {formik.values?.spec?.updateMultipleFlag ? (
+                <>
+                  <FormInput.MultiTextInput
+                    tooltipProps={{
+                      dataTooltipId: 'serviceNowApprovalChangeRequestNumber'
+                    }}
+                    label={getString('pipeline.serviceNowApprovalStep.changeRequestNumber')}
+                    name={`spec.updateMultiple.spec.changeRequestNumber`}
```

Comment:
```suggestion
                    name={`spec.updateMultiple.changeRequestNumber`}
```
Make sure name matches with the interface.